pkg/api/kptfile/v1/types.go

```diff
@@ -29,6 +29,10 @@ import (
 const (
 	KptFileName = "Kptfile"
 
+	RevisionMetaDataFileName = "KptRevisionMetaData"
```

Comment:
What is the rationale behind this name? It seems somewhat long, but I'm not what would be a better name. We could also consider if we should use a hidden file for this (some something that starts with `.`.

Reply:
As the content in this file is specific to package revision and it will represent the meta data named it as KptRevisionMetaData. If its too long we can name it as KptRevisionFile. Let me know if any other name suits better.

We can make it a hidden file too, let me know if we want do that

Reply:
Ok, so let's think about some pros and cons:
- If it's visible, then users may mess with it. At the very least, they probably would expect it to let them make changes to the metadata. For example, can they add any annotations they want there? Does it only affect PackageRevisionResources or would it affect PackageRevision as well? Maybe we want to allow that? It's more to implement, though.
- If it's not visible, then we can worry less about users messing with it or about setting some expectations. However, it makes it less transparent how things work.

I guess on balance I would recommend:
- We hide it; the name is fine but add the "." to hide it.
- In the "push" command, if we cannot fine the file, we output a very clear message: ".KptRevisionMetaData file not found. This file is created by the pull command and is required by the push command", or something similar.

Reply:
Sure it makes more sense to hide the file, I will update it